pkgs/development/python-modules/coloraide/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+  typing-extensions,
+}:
+let
+  pname = "coloraide";
+  version = "4.7.2";
+in
+buildPythonPackage {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-fomOKtF3hzgJvR9f2x2QYYrYdASf6tlS/0Rw0VdmbUs=";
+  };
+
+  build-system = [
+    hatchling
+  ];
+
+  dependencies = [
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [
+    "coloraide"
+  ];
+
+  meta = {
+    description = "A color library for Python";
```

```suggestion
    description = "Color library for Python";
```
Not start with the definite or an indefinite article.

---

Library descriptions are often automatically generated from upstream, as this one is. The Haskell package set uses the descriptions from upstream as well:

```
$ nix eval -f . haskellPackages.Agda.meta.description
"A dependently typed functional programming language and proof assistant"
````

---

Well the haskell packages set is generated automatically, it would be time-consuming and unreasonable to edit every description by hand. Since python packages are manually generated, it is much easier to update these to fit our conventions.

---

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes